src/main/scala/diplomaticobjectmodel/model/OMCoreComplex.scala

```diff
@@ -2,8 +2,27 @@
 
 package freechips.rocketchip.diplomaticobjectmodel.model
 
+case class CoreComplexRTLInterface(
+  clocks: List[OMClock] = Nil,
+  clockRelationships: List[OMClockRelationship] = Nil,
+  resets: List[OMRTLReset] = Nil,
+  statuses: List[OMStatus] = Nil,
+  localInterrupts: Option[OMInterruptSignal] = None,  // E.g. local_interrupts_X (all local interrupts for core X)
+  globalInterrupts: Option[OMInterruptSignal] = None,  // E.g. global_interrupts
+  machineExternalInterrupts: Option[OMInterruptSignal] = None,  // E.g. meip_X
```

Comment:
These should be Seqs, not Options. I believe the current implementation will have one instance of the local and the machine external interrupts per core. The global interrupts might be a singleton.